src/App.tsx

```diff
@@ -1,87 +1,48 @@
-import React, { useEffect } from 'react';
-import logo from './logo.svg';
 import './App.css';
-import { useRef, useState } from 'react'
-
-interface Dimensions {
-  h: number;
-  w: number;
-}
-
-function computeFrame() {
-  
-}
+import { useRef, useState, useEffect, useCallback } from 'react';
 
 function App() {
-  const [videoDimensions, setVideoDimensions] = useState<Dimensions>({ h: 0, w: 0 })
-  const [canvasDimensions, setCanvasDimensions] = useState<Dimensions>({ h: 0, w: 0})
-  const [counter, setCounter] = useState(0);
```

Comment:
switching the number for a boolean we can toggle to make sure we don't int overflow for very long videos